central/complianceoperator/v2/scanconfigurations/service/convert.go

```diff
@@ -46,11 +45,10 @@ func convertStorageScanConfigToV2(ctx context.Context, scanConfig *storage.Compl
 		clusters = append(clusters, cluster.GetClusterId())
 	}
 
-	// TODO(ROX-18102):  See if we want to lookup profile name by id instead of storing it
 	var profiles []string
```

Comment:
Presize this slice and remove the subsequent intermediate var (scanConfig.GetProfiles()) as it's only used in one place